query-service-impl/src/main/java/org/hypertrace/core/query/service/postgres/PostgresBasedRequestHandler.java

```diff
@@ -60,6 +60,8 @@ public class PostgresBasedRequestHandler implements RequestHandler {
   private static final int DEFAULT_SLOW_QUERY_THRESHOLD_MS = 3000;
   private static final Set<Operator> GTE_OPERATORS = Set.of(Operator.GE, Operator.GT, Operator.EQ);
 
+  private static final Value NULL_VALUE = Value.newBuilder().setString("null").build();
```

Comment:
Note: The `null` string is set in QS o/p in other handlers, so this fix will align postgres impl.